http/src/main/java/io/hyperfoil/http/connection/Http1xConnection.java

```diff
@@ -366,7 +371,9 @@ public void putHeader(CharSequence header, CharSequence value) {
          }
          buf.writeByte('\r');
          buf.writeByte('\n');
-         HttpCache.get(request.session).requestHeader(request, header, value);
+         if (request.hasCacheControl()) {
```

Comment:
if the httpCache is not null it should always have the cache control (as it was before?)

Reply:
yeah that's true, do you see something wrong here?